jobs/build/build-sync/Jenkinsfile

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    def commonlib = buildlib.commonlib
+
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '360'
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    commonlib.mockParam(),
+		    commonlib.ocpVersionParam('BUILD_VERSION', '4'),
+                ]
+            ],
+            disableConcurrentBuilds(),
+            timestamps(),
+        ]
+    )
+
+    echo "Initializing ${params.BUILD_VERSION} sync: #${currentBuild.number}"
+
+    // doozer_working must be in WORKSPACE in order to have artifacts archived
+    def mirrorWorking = "${WORKSPACE}/MIRROR_working"
+    // Location of the SRC=DEST input file
+    def ocMirrorInput = "${mirrorWorking}/oc_mirror_input"
+    // Location of the image stream to apply
+    def ocIsObject = "${mirrorWorking}/release-is.yaml"
+    // Locally stored image stream stub
+    def baseImageStream = "/home/jenkins/base-art-latest-imagestream.yaml"
+    // Kubeconfig allowing ART to interact with api.ci.openshift.org
+    def ciKubeconfig = "/home/jenkins/kubeconfigs/art-publish.kubeconfig"
+
+    // See 'oc image mirror --help' for more information.
+    // This is the template for the SRC=DEST strings mentioned above.
+    def ocFmtStr = "registry.reg-aws.openshift.com:443/{repository}=quay.io/openshift-release-dev/ocp-v4.0-art-dev:{version}-{release}-{image_name_short}"
+
+    //Clear out previous work
+    sh "rm -rf ${mirrorWorking}"
+    sh "mkdir -p ${mirrorWorking}"
+
+    stage("Version dumps") {
+        buildlib.doozer """
+--version
+"""
```

Comment:
why multiline?

Reply:
Consistency. Which I have broken. Will update to be consistent!